### PR TITLE
refactor: make TWAP warning less scary

### PR DESCRIPTION
**Pull request by compojoom**
# Summary
The TWAP warning was very technical and "scary" to a normal user. I've modified it such that it would be more understandable but a normal human being without going into the technicalities (fallbackhandler).

# To Test

1. Create a twap order on a safe that doesn't have cow's fallbackhandler. The message you see should look like this:
grafik

### CODE REVIEW

File: apps/cowswap-frontend/src/modules/twap/containers/TwapFormWarnings/warnings/FallbackHandlerWarning.tsx [hunk 1]
```diff
@@ -81,10 +82,10 @@ export function FallbackHandlerWarning({
     return (
       <Wrapper>
         <InlineBannerWithCheckbox bannerType="alert">
-          <strong>Unsupported Safe detected</strong>
+          <strong>Your Safe needs a modification</strong>
           <p>
-            Connected Safe lacks required fallback handler. Switch to a compatible Safe or modify fallback handler for
-            TWAP orders when placing your order.
+            TWAP orders require a one-time update to your Safe to enable signing of conditional (e.g. time-based)
+            orders. It allows them to be automatically executed in the future, once the original TWAP is approved.
```
Comment (anxolin) [inline apps/cowswap-frontend/src/modules/twap/containers/TwapFormWarnings/warnings/FallbackHandlerWarning.tsx:88]: One suggestion would be:

```suggestion
            TWAP orders require a one-time update to your Safe to enable programmatic orders.  These advanced orders allow you to automate your trading strategy with on-chain guarantees. For more information, see [Programmatic orders](https://cow.fi/learn/introducing-the-programmatic-order-framework-from-cow-protocol).
```

**Comment by kirkkonen** [comment]
Guys, just please change the text to "TWAP orders require a one-time update to your Safe to enable automated execution of scheduled transactions". The current one and the suggested one is confusing, long, too technical and unclear.

The order is a mere scheduled execution. User doesn't have to know anything else. If they do they can "read more".

Don't overcomplicate these things please (both the messaging and the process behind its changing).

This is Safe{Wallet} UI and we need the communication with the user to be clear.

**Comment by anxolin** [comment]
Above is my suggestion, but happy to pick the message @mindy feels happy with

**Comment by anxolin** [comment]
I can explain why I chose the options above 

I tried to get away of the "modification" word in "Make the modification when placing order". It sounds scary. `Enable programmatic order` sounds more positive cause you are enabling a feature


`Programmatic order` sounds technical, so does account abstraction and many other concepts we manage. Marketing chose that term to refer to the framework and some users might love to be able to see some context.

Having said that, I don't care too much on the message above, delegating my decision to @MindyCoW

**Comment by MindyCoW** [comment]
I'm okay with Andrei's proposed message. It's concise and clear.

**Comment by compojoom** [comment]
Ok, in this case my latest commit should represent Andrei's wish for text.

**Comment by elena-zh** [comment]
Hi there!
A bit confused with all these discussions, so I'd rather clarify: is this message acceptable to everyone? 
image
IMO, it became longer than the previous one. 
image

**Comment by compojoom** [comment]
@elena-zh, sorry I forgot to update the PR description. This is currently the last version:
grafik

**Review by elena-zh — APPROVED** [review]
Approved then :)

**Review by alfetopito — APPROVED** [review]

**Pull request merged**